Jenkinsfile

```diff
@@ -19,8 +19,9 @@ node('imf2public') {
                      bat '''
                     set TEMP=%WORKSPACE%
                     set TMP=%TEMP%
-                    
-                    %PYTHONPATH%python.exe ago-group-update.py -user %agouser% -pwd %agopassword% -group %group%
+                    set AGOPASSWORD=jenkins.model.Jenkins.instance.getCredentials().getById("d9d435e2-8acb-4698-bc74-2a1729472f1a").getPassword()
```

Comment:
Does this need to be an environment variable? I'd assume that exposes the pwd in the system running this - if someone else knew the name of the env var they might be able to get its value. A local variable would be safer.

Reply:
Firstly, sorry I should have squashed these commits, I didn't use this code - I actually went about it this way - https://github.com/bcgov/ago-all-gov-group-update-tool/pull/7/commits/7d4dc0d67a816cff0139e2bfac1ec79131f64b21

2nd, good question. The password is still an environment variable there, but I'm not sure how accessible it is. e.g. echo %AGOPASSWORD% in the batch script just sent ***** to the console